Khan/wonder-blocks
# Add a story to demonstrate long option labels

## Summary:
Add a story to demonstrate the long option labels
become truncated with trailing ellipses.

Inspired by this [Slack thread](https://khanacademy.slack.com/archives/C8Z9DSKC7/p1678212891180559).

Issue: none

## Test plan:
- Go to http://localhost:49249/?path=/story/dropdown-singleselect--long-option-labels
- Confirm that the opener label is truncated with an ellipsis
- Open the dropdown and confirm that all the option items are
  truncated with ellipses

<img width="1090" alt="image" src="https://user-images.githubusercontent.com/13231763/223593887-53fb3135-2fd2-40f0-8cfe-40e422f78dd4.png">
